packages/stryker-webpack-transpiler/src/compiler/WebpackCompiler.ts

```diff
@@ -0,0 +1,66 @@
+import { TextFile, File, BinaryFile, FileKind } from 'stryker-api/core';
+import { Compiler, Configuration } from 'webpack';
+import webpack from './Webpack';
+import * as path from 'path';
+import InputFileSystem from '../fs/InputFileSystem';
+import OutputFileSystem from '../fs/OutputFileSystem';
+
+export default class WebpackCompiler {
+  private _compiler: Compiler;
+  private _inputFS: InputFileSystem;
+  private _outputFS = new OutputFileSystem();
+
+  public constructor(webpackConfig: Configuration) {
+    this._compiler = this.createCompiler(webpackConfig);
+  }
+
+  private createCompiler(webpackConfig: Configuration): Compiler {
+    const compiler = webpack(webpackConfig);
+    this._inputFS = new InputFileSystem();
```

Comment:
Can't you pass an initialized `InputFileSystem` in the constructor, thereby using dependency injection? That would also make it easier to mock away the filesystem in the unit tests.